server.go

```diff
@@ -95,6 +97,22 @@ func runMain() error {
 		return fmt.Errorf("Failed to initialize the smtp server: %v", err)
 	}
 
+	// get the default master extended public key
```

Comment:
dcrstakepool repo might have different standards, but we generally want comments that are written as sentences to begin with a capital letter and end with punctuation.

Reply:
Maybe also reword this as "Retrieve the extended public key for the voting wallet's "default" account."

Reply:
No reason for this repo to be any different, we should standardise across all (when it is reasonable to do so)